app/assets/stylesheets/views/article-form.scss

```diff
@@ -174,6 +174,10 @@
     > :first-child {
       margin-left: calc(var(--su-2) * -1);
     }
+
+    @media (max-width: $breakpoint-m) {
+      padding-right: 0;
+    }
```

Comment:
For the toolbar, `&__toolbar`, I imagine we can use a utility class like `m:p-0` instead on the tag?

Reply:
I had a look at this, but it's tricky because those responsive helpers are designed to take effect from the given breakpoint, but I want to target anything _until_ that breakpoint (e.g. `m:p-0` would do the opposite of what I want to do).

The padding for this toolbar is already declared in this file so we can use the `--content-padding` variable, so it seemed best to keep this styling together.

Reply:
How about this... May seem over-engineering at first glance, but I think it's worth for keeping min-widths over max-widths for our breakpoints.. Thoughts? (also, I haven't tested it :D)

```
&__toolbar {
  padding: var(--su-2) var(--content-padding-x);
  padding-right: var(--toolbar-padding-right, 0);

  @media (min-width: $breakpoint-m) {
    --toolbar-padding-right: var(--content-padding-x);
  }
}
```

Reply:
That sounds like a good plan 🙌